include/xvega/utils/xmaterialize.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020, QuantStack and XVega Contributors
+//
+// Distributed under the terms of the BSD 3-Clause License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+
+#ifndef XVEGA_MATERIALIZE_HPP
+#define XVEGA_MATERIALIZE_HPP
+
+#include <utility>
+
+#include "nlohmann/json.hpp"
+
+#include "../xvega_config.hpp"
+
+namespace nl = nlohmann;
```

Comment:
I should have removed the references to `nlohmann_json`, 

- `namespace nl = nlohmann;`
- `#include "nlohmann/json.hpp"`